### DISCUSSION TITLE
QTreeAPI behavior: classifier

### DISCUSSION THREAD

**Discussion by Zaker237** [quasarframework/quasar#9992] [General - Components / Directives / etc]
**Is your feature request related to a problem? Please describe.**
in my project i use the QTree to show the folder content and i wanted the tree's content to be in alphabetic order.

**Describe the solution you'd like**
it will be nice to have the option that we can set to true to automatic classify the content of the tree. and may be an additional option to set the order(ascending or descending).

**Describe alternatives you've considered**

**Additional context**
we call also set the attritut on which we want classify, in my case a maded it on label.

**Comment by pdanpdan** [comment]
Prepare the data for the tree before. It's very easy.
If you don't know how to do it make a codepen with what you have tried and someone will help
